### PR TITLE
Add HealthCheckCompletedWebhookEvent

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
### Prerequisites

- [x] I have added steps to test this contribution in the description below

### Description
In addition to the "Webhook all the things" [PR](https://github.com/umbraco/Umbraco-CMS/pull/15161) and as suggested by @nul800sebastiaan [in](https://github.com/umbraco/Umbraco-CMS/pull/15276#issuecomment-1833936881) add a `HealthCheckCompletedWebhookEvent`. 

Add to the collectionbuilder: 

```csharp
builder.WebhookEvents().AddHealthCheckWebhooks();
```

or:
```csharp
builder.WebhookEvents().AddAllAvailableWebhooks();
```

### CODE REVIEW

**Comment by emmagarland** [comment]
Hi @erikjanwestendorp ,

Thanks for your PR to add a HealthCheckCompletedWebhookEvent as part of the overall ["Webhook all the things"](https://github.com/umbraco/Umbraco-CMS/pull/15276#issuecomment-1833936881) strategy!

We'll have a gander soon. 

Tagging @nul800sebastiaan since this is targeting a branch we can't merge.

Best wishes

Emma

File: src/Umbraco.Core/Webhooks/Events/HealthCheck/HealthCheckCompletedWebhookEvent.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+using Microsoft.Extensions.Options;
+using Umbraco.Cms.Core.Configuration.Models;
+using Umbraco.Cms.Core.Notifications;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.Core.Sync;
+
+namespace Umbraco.Cms.Core.Webhooks.Events.HealthCheck;
```
Comment (ronaldbarendse) [inline src/Umbraco.Core/Webhooks/Events/HealthCheck/HealthCheckCompletedWebhookEvent.cs:7]:
```suggestion
namespace Umbraco.Cms.Core.Webhooks.Events;
```

File: src/Umbraco.Core/Webhooks/WebhookEventCollectionBuilderCmsExtensions.cs [hunk 2]
```diff
@@ -313,6 +314,17 @@ public static WebhookEventCollectionBuilderCms AddRelationType(this WebhookEvent
...
+    /// <summary>
+    /// Adds the healthcheck webhook events.
+    /// </summary>
+    /// <param name="builder">The builder.</param>
+    /// <returns>
+    /// The builder.
+    /// </returns>
+    public static WebhookEventCollectionBuilder AddHealthCheck(this WebhookEventCollectionBuilder builder)
+        => builder
+            .Append<HealthCheckCompletedWebhookEvent>();
```
Comment (ronaldbarendse) [inline src/Umbraco.Core/Webhooks/WebhookEventCollectionBuilderCmsExtensions.cs:326]: This should use the `WebhookEventCollectionBuilderCms` builder!

File: src/Umbraco.Core/Webhooks/WebhookEventCollectionBuilderExtensions.cs [hunk 3]
```diff
@@ -38,7 +38,8 @@ public static WebhookEventCollectionBuilder AddCms(this WebhookEventCollectionBu
                     .AddPublicAccess()
                     .AddRelation()
                     .AddRelationType()
-                    .AddUser();
+                    .AddUser()
+                    .AddHealthCheck();
```
Comment (ronaldbarendse) [inline src/Umbraco.Core/Webhooks/WebhookEventCollectionBuilderExtensions.cs:42]: And lets order this alphabetically!

**Comment by nul800sebastiaan** [comment]
You were too soon with your update @erikjanwestendorp - Ronald had moved things again in #15345 haha! I fixed the resulting conflict though and this one is now good to go! Thanks for staying on top of it!

**Review by ronaldbarendse — CHANGES_REQUESTED** [review]

**Comment by nul800sebastiaan** [comment]
Even worse, I broke the build! But thanks @ronaldbarendse for the quick fix! 8a65c002793eefc942f5e289b247fd2d3df0d01a

**Pull request merged**